tests/unit/syslog_monitor_test.py

```diff
@@ -23,7 +22,7 @@
 if False:
     from typing import List
 
-__author__ = "imron@scalyr.com"
+__author__ = "scalyr-cloudtech@scalyr.com"
```

Comment:
I would change this to `scalyr@sentinelone.com`

Reply:
Will do